citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.html

```diff
@@ -1147,7 +1147,7 @@ <h2>
 							&nbsp;&nbsp;
 						</th:block>
 					</span>
-						<div class="pull-right"><a data-toggle="modal" data-target="#authorModal"><i class="icon-circle-add"></i> Add Author</a></div>
+						<div class="pull-right"><a data-toggle="modal" data-target="#authorModal"><i class="icon-circle-add" style="cursor: pointer; color: #337ab7"></i> Add Author</a></div>
```

Comment:
that shouldn't have to be changed for every single icon; there should be a general fix for this.

Reply:
should I be changing diging-icon-pack for a general fix for this